src/pages/BestList/BestList.js

```diff
@@ -1,58 +1,34 @@
 import React from 'react';
 
+import BannerTitle from './BannerTitle/BannerTitle';
+import BannerList from './BannerList/BannerList';
+
 import './BestList.scss';
 
 class BestList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      storeList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://10.58.0.218:8000/restaurants?ordering=average_rating')
```

Comment:
api 주소 src/config.js 경로 등에 분리해서 관리해주세요
```suggestion
    fetch(`${API}/restaurants?ordering=average_rating`)
```

Reply:
한번에 정리하려고합니다...